data-raw/instream/clear/clear_creek.rmd

```diff
@@ -22,28 +22,26 @@ Scale spawning wua, examine emails from Mark Gard and Charile Chamberline to try
 **Data Source:**     
```

Comment:
Change Future Data Improvements to None planned, remove stuff about scaling spawning wua